packages/util/src/object.ts

```diff
@@ -98,6 +98,25 @@ export const pick = <T>(keys: string[], obj: T): Partial<T> => {
   return result;
 };
 
+/**
+ * Create a shallow copy of the given object, that does not contain the given keys.
+ * Non existing keys in the source object are ignored.
+ *
+ * @param keys - properties to be selected
+ * @param obj - object from which the values are taken
+ * @returns an object with the selected keys and corresponding values.
+ */
+export const pickWithoutKeys = <T>(keys: string[], obj: T): Partial<T> => {
```

Comment:
Consider `exclude` as a name. "Pick without keys" sounds like there might be no keys in the result (so an array?) ;)
This also correlates with the naming of the according TypeScript types.